DEVELOPMENT.md

```diff
@@ -44,14 +44,54 @@ To spin up a development environment for the *jenkins-datadog* plugin repository
           - 8080:8080
         volumes:
           - $JENKINS_PLUGIN/target/:/var/jenkins_home/plugins
+    ## Uncomment environment variables based on your needs. Everything can be configured in jenkins /configure page as well. 
+    #      - DATADOG_JENKINS_PLUGIN_REPORT_WITH=DSD
+    #      - DATADOG_JENKINS_PLUGIN_COLLECT_BUILD_LOGS=false
+    ## Set `DATADOG_JENKINS_PLUGIN_TARGET_HOST` to `dogstatsd` or `datadog` based on the container you wish to use.
+    #      - DATADOG_JENKINS_PLUGIN_TARGET_HOST=dogstatsd
+    #      - DATADOG_JENKINS_PLUGIN_TARGET_LOG_COLLECTION_PORT=10518
+    #      - DATADOG_JENKINS_PLUGIN_TARGET_API_KEY=$JENKINS_PLUGIN_DATADOG_API_KEY
+      
+    ## Uncomment the section below to use the standalone DogStatsD server to send metrics to Datadog
+    #  dogstatsd:
+    #    image: datadog/dogstatsd:latest
+    #    environment:
+    #      - DD_API_KEY=$JENKINS_PLUGIN_DATADOG_API_KEY
+    #    ports:
+    #      - 8125:8125
+    
+    ## Uncomment the section below to use the whole Datadog Agent to send metrics (and logs) to Datadog. 
+    ## Note that it contains a DogStatsD server as well.
+    #  datadog:
+    #    image: datadog/agent:latest
+    #    environment:
+    #      - DD_API_KEY=$JENKINS_PLUGIN_DATADOG_API_KEY
+    #      - DD_LOGS_ENABLED=true
+    #     ports:
+    #       - 8125:8125
+    #       - 10518:10518
+    #    volumes:
+    #      - /var/run/docker.sock:/var/run/docker.sock:ro
+    #      - /proc/:/host/proc/:ro
+    #      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+    #      - $JENKINS_PLUGIN/conf.yaml:/etc/datadog-agent/conf.d/jenkins.d/conf.yaml   
                
     ```
+1. If you wish to submit log using the Datadog Agent, you will have to configure the Datadog Agent properly by creating a conf.yaml file with the following content.
```

Comment:
Let's call it `datadog.yaml` for consistency
```suggestion
1. If you wish to submit log using the Datadog Agent, you will have to configure the Datadog Agent properly by creating a `datadog.yaml` file with the following content.
```

Reply:
It is not the datadog.yaml file. it is a conf.yaml file. See https://docs.datadoghq.com/agent/logs/?tab=recommendedhttpcompressed#custom-log-collection

Reply:
oh right. Let's make it more explicit then, that it should be a conf.yaml in the conf.d folder. That link to the doc would be a great addition as well

Reply:
Isn't it good enough with the volume mounting param in the `docker-compose` file?